<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>604</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>dat2png</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Celeste/Pictures/lol.ico</normaloff>C:/Users/Celeste/Pictures/lol.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>550</height>
    </size>
   </property>
   <widget class="QWidget" name="info_wgt" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1001</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QLabel" name="lbl_tip1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>dat文件目录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="text1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>811</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>961</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_infoshow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string> 信息面板</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exec">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>410</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="text_area1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>961</width>
       <height>150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'等线','黑体'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1、选择*.dat文件的初始目录与 *.png文件的输出目录 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;2、输入您的&lt;/span&gt; &lt;span style=&quot; font-family:'SimSun'; font-weight:600; color:#000000;&quot;&gt;异或参数值&lt;/span&gt; 来获取转换文件（异或参数值联系管理员由计算得出，每个Wechat用户的异或参数值有所不同）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_tip2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>文件输出目录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="text2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>811</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_tip3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>异或参数值</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="text3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>0x80</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>961</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>关于 </string>
    </property>
    <addaction name="act_version"/>
    <addaction name="act_exit"/>
   </widget>
   <addaction name="menu_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_createfile">
   <property name="text">
    <string>创建文件</string>
   </property>
  </action>
  <action name="act_openfile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="act_savefile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="act_run">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="act_debug">
   <property name="text">
    <string>调试</string>
   </property>
  </action>
  <action name="act_version">
   <property name="text">
    <string>版本</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="act_param">
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
  <action name="act_tips">
   <property name="text">
    <string>提示</string>
   </property>
  </action>
  <action name="act_mail">
   <property name="text">
    <string>联系</string>
   </property>
  </action>
  <action name="act_graphene">
   <property name="text">
    <string>Graphene</string>
   </property>
  </action>
  <action name="act_BN">
   <property name="text">
    <string>BN</string>
   </property>
  </action>
  <action name="act_MoS2">
   <property name="text">
    <string>MoS2</string>
   </property>
  </action>
  <action name="act_forbid">
   <property name="text">
    <string>此项不可选择</string>
   </property>
  </action>
  <action name="act_log">
   <property name="text">
    <string>系统日志</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
